aiplatform/src/test/java/aiplatform/BatchCodePredictionSampleTest.java

```diff
@@ -61,38 +60,26 @@ public static void checkRequirements() throws IOException {
     BUCKET_NAME = "my-new-test-bucket" + UUID.randomUUID();
 
     // Create a Google Cloud Storage bucket for UsageReports
-    Storage storage = StorageOptions.newBuilder().setProjectId(PROJECT_ID).build().getService();
+    storage = StorageOptions.newBuilder().setProjectId(PROJECT_ID).build().getService();
     storage.create(BucketInfo.of(BUCKET_NAME));
   }
 
   @AfterClass
   public static void afterClass() {
     // Delete the Google Cloud Storage bucket created for usage reports.
-    Storage storage = StorageOptions.newBuilder().setProjectId(PROJECT_ID).build().getService();
-    Bucket bucket = storage.get(BUCKET_NAME);
+    storage = StorageOptions.newBuilder().setProjectId(PROJECT_ID).build().getService();
+    bucket = storage.get(BUCKET_NAME);
     bucket.delete();
   }
 
-  @Before
-  public void beforeEach() {
-    stdOut = new ByteArrayOutputStream();
-    System.setOut(new PrintStream(stdOut));
-  }
-
-  @After
-  public void afterEach() {
-    stdOut = null;
-    System.setOut(null);
-  }
-
   @Test
   public void testBatchCodePredictionSample() throws IOException {
 
-    BatchCodePredictionSample.batchCodePredictionSample(
-        PROJECT_ID, LOCATION, GCS_SOURCE_URI,
+    BatchPredictionJob batchPredictionJob =
+        BatchCodePredictionSample.batchCodePredictionSample(PROJECT_ID, LOCATION, GCS_SOURCE_URI,
         GCS_DESTINATION_OUTPUT_PREFIX, MODEL_ID);
 
-    assertThat(stdOut.toString()).contains("publishers/google/models/code-bison");
-    assertThat(stdOut.toString()).contains("my batch code prediction job");
+    Assertions.assertNotNull(batchPredictionJob);
+    assert (batchPredictionJob.getState().equals(JOB_STATE_PENDING));
```

Comment:
This may or may not be true, depending upon the state of the process. I would suggest to remove this assertion.